docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -0,0 +1,391 @@
+# Defining placement rules in SQL
+
+- Author(s):     [djshow832](https://github.com/djshow832) (Ming Zhang)
+- Last updated:  2020-06-24
+- Discussion at: https://docs.google.com/document/d/18Kdhi90dv33muF9k_VAIccNLeGf-DdQyUc8JlWF9Gok
+
+## Movitation
+
+TiDB supports placement rules, which can define the placement of data in a more flexible and more granular way. But it only provides configuration files to define them, and it’s complicated.
+
+This article proposes an approach to configure placement rules through statements. TiDB server parses the statements and notify PD to perform the change. In this way, usability can be improved.
+
+## Define placement rules
+
+There are 2 kinds of operations on the placement:
+
+* ADD: Add new `count` replicas to the table according to `label` and `role` configuration.
+* ALTER: Override the current replica configuration. Old configuration is discarded.
+
+They’re both achieved by executing `ALTER TABLE` statements.
```

Comment:
Not yet. It would be much more complicated. But for some serious scenarios such as data sovereignty, it's a must. I'm consider supporting it in the next phase.